jest.config.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  projects: ['<rootDir>/packages/*/jest.config.ts'],
```

Comment:
jest monorepo configuration. It will search for the packages folder for every jest.config file and collect the coverage for specified regexs